test/MockServer.hs

```diff
@@ -59,6 +59,7 @@ mockMCServer loop resp' sem = forkIO $ bracket
         let hints = N.defaultHints {
             N.addrFlags = [N.AI_PASSIVE]
           , N.addrSocketType = N.Stream
+          , N.addrFamily = N.AF_INET
```

Comment:
This was useful because `getAddrInfo` will return both ipv4 and ipv6, and will cause things to fail if the family and the address don't agree. This hint makes it more likely we'll have ipv4 across them.